docs/guides/cloud/organizations.mdx

```diff
@@ -119,44 +77,44 @@ your Organization must meet the following criteria:
 
 ### Requesting OSS plan for an Org
```

Comment:
As just a thought, the OSS plan info seems out of place in Billing & Usage, especially since there isn't any actual info on Billing & Usage in this section. I think OSS could be its own header, or even in a different guide.

Reply:
Yeah it's a little funny. You only get to this by way of Billing & Usage which seems like it iframes in a new page. I have a separate PR to reorganize content and give Billing & Usage more emphasis.